fs-state/init_globals.h

```diff
@@ -51,12 +51,12 @@ typedef struct all_dev_nums {
 static const char *fs_all[] = {"btrfs", "ext2", "ext4", "f2fs", 
                                "jffs2", "ramfs", "tmpfs", "verifs1", 
                                "verifs2", "xfs", "nilfs2", "jfs",
-                               "nova"};
+                               "nova", "nfs"};
                                
 static const char *dev_all[]= {"ram", "ram", "ram", "ram", 
                                 "mtdblock", "", "", "", 
                                 "", "ram", "ram", "ram",
-                                "pmem"};
```

Comment:
Currently for nfs-ext4, this approach works. 
To expand for using nfs with other file systems, this requires a more careful approach. 
Maybe we can create an extra variable in globals.h to specify the underlying fs so that we can remove this dependency on ramdisk only